### PR TITLE
Added assignment-from-no-return message example

### PR DESCRIPTION
**Pull request by matusvalo**
## Type of Changes

|     | Type                   |
| --- | ---------------------- |
|     | :bug: Bug fix          |
|     | :sparkles: New feature |
|     | :hammer: Refactoring   |
| ✓   | :scroll: Docs          |

## Description

Closes partially #5953

### CODE REVIEW

File: doc/data/messages/a/assignment-from-no-return/bad.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+def add(x, y):
+    print(x + y)
+
+
+value = add(10, 10)  # [assignment-from-no-return]
+print(value)
```
Comment (DanielNoord) [inline doc/data/messages/a/assignment-from-no-return/bad.py:6]:
```suggestion
```

File: doc/data/messages/a/assignment-from-no-return/good.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+def add(x, y):
+    return x + y
+
+
+value = add(10, 10)
+print(value)
```
Comment (DanielNoord) [inline doc/data/messages/a/assignment-from-no-return/good.py:6]:
```suggestion
```

**Review by DanielNoord — APPROVED** [review]
Bit nit picky, but let's try to keep this examples as short as possible. For a beginner, every additional line might be interpreted as necessary for the code to work or the message to go away.

**Comment by matusvalo** [comment]
> Bit nit picky, but let's try to keep this examples as short as possible.

I put print to leave example having just one single error - the one which is documented. (if print is missing there is error about not used variable). But this is just matter of taste.

**Comment by DanielNoord** [comment]
> > Bit nit picky, but let's try to keep this examples as short as possible.
> 
> I put print to leave example having just one single error - the one which is documented. (if print is missing there is error about not used variable). But this is just matter of taste.

I agree with the principle but I think it will be unmanageable. For stuff which involves classes we would need to satisfy `too-few-public-methods`, all the `missing-docstring` messages, the `no-self-use` etc.
At that point we might as well write a complete module 😄 

The tester has been specifically set up to only enable the message that is in the name of the directory so any other pylint messages are no issue for these.

**Review by Pierre-Sassoulas — APPROVED** [review]

**Pull request merged**